src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const headers = [...document.querySelector('thead').children[0].children];
+const tbody = document.querySelector('tbody');
+const peopleRows = [...document.querySelector('tbody').children];
+
+headers.forEach(head => {
+  const index = headers.indexOf(head);
```

Comment:
The 'indexOf' method is not efficient here, use the second parameter of 'forEach' to get the index.